Sources/TuistLoader/Utils/PluginsTemplatePathHelper.swift

```diff
@@ -0,0 +1,55 @@
+import Foundation
+import TSCBasic
+import TuistCore
+import TuistGraph
+import TuistSupport
+
+protocol PluginsTemplatePathHelping {
```

Comment:
We have other utilities with a similar purpose named `...DirectoryLocator`. I'd be consistent with that to make it easier to find these utilities in the codebase:

![Uploading image.png…]()

Reply:
good point, I'll rename it